backend/cfg/cfg_invariants.ml

```diff
@@ -309,6 +309,12 @@ let check_block t label (block : Cfg.basic_block) =
   check_stack_offset t label block;
   ()
 
+let check_reducibility t cfg_with_layout =
+  let cfg_with_infos = Cfg_with_infos.make cfg_with_layout in
```

Comment:
This is going to fail when cfg_invariants is called at the end of the compilation pipeline, after "allow_irreducible" is set.